experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -166,28 +206,50 @@ export const makeFieldEditGenerator = (
 		}
 	}
 
-	function generateSequenceFieldDeleteOp(
+	function generateDeleteEdit(
 		fieldPath: FieldUpPath,
-		random: IRandom,
 		count: number,
 		treeIndex: number,
-	): SequenceFieldEdit {
-		const nodeIndex = random.integer(0, count - 1);
-		const rangeSize = random.integer(1, count - nodeIndex);
+		nodeIndex: number,
+	): FuzzDelete {
 		const firstNode: UpPath = {
 			parent: fieldPath.parent,
 			parentField: fieldPath.field,
 			parentIndex: nodeIndex,
 		};
-		const contents: FuzzDelete = {
+		return {
 			type: "delete",
 			firstNode,
-			count: rangeSize,
+			count,
 			treeIndex,
 		};
+	}
+
+	function generateSequenceFieldDeleteOp(
+		fieldPath: FieldUpPath,
+		random: IRandom,
+		count: number,
+		treeIndex: number,
+	): SequenceFieldEdit {
+		const nodeIndex = random.integer(0, count - 1);
+		const rangeSize = random.integer(1, count - nodeIndex);
+		const contents = generateDeleteEdit(fieldPath, rangeSize, treeIndex, nodeIndex);
 		return { type: "sequence", edit: contents };
 	}
 
+	function generateValueFieldDeleteOp(fieldPath: FieldUpPath, treeIndex: number): ValueFieldEdit {
+		const contents = generateDeleteEdit(fieldPath, 1, treeIndex, 0);
```

Comment:
nit: name this `edit`, or inline it in the line below

Same for function below